packages/password-field/theme/material/vaadin-password-field-button-styles.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { button } from '@vaadin/button/theme/material/vaadin-button-styles.js';
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+const passwordFieldButton = css`
+  :host {
+    position: absolute;
+    right: 0;
+    top: 0;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
+    min-width: auto;
+    background: transparent;
+    outline: none;
+    border-radius: 50%;
+    overflow: visible;
+  }
+
+  :host::before {
+    transform: scale(1.5);
+  }
+
+  /* Disable ripple */
+  :host::after {
+    display: none;
+  }
```

Comment:
I had to set `overflow: visible` in order to let the `scale(1.5)` styles work properly.
This caused the ripple to be very wide, so for now I decided to disable it 🙈